src/common/components/Input/style.css.ts

```diff
@@ -95,3 +97,12 @@ export const descriptionVar = styleVariants(
     },
   ],
 );
+
+export const timer = style({
+  position: 'absolute',
+  right: 172,
+  top: 16,
```

Comment:
이렇게 해도 되지만 정확히 딱 중앙에 맞추고 싶다면 아래의 방법을 이용하시면 됩니다~
```suggestion
  top: '50%',
  transform: 'translateY(-50%)',
```

Reply:
너무 좋네요!